setup-test-env.js

```diff
@@ -1,5 +1,7 @@
 /* global expect */
-import { createSerializer } from 'jest-emotion';
+import { createSerializer, matchers } from 'jest-emotion';
 import * as emotion from '@emotion/core';
+import '@testing-library/jest-dom';
 
+expect.extend(matchers);
```

Comment:
what is `matchers` providing here?

Reply:
This makes all the [`jest-dom` matchers](https://github.com/testing-library/jest-dom#custom-matchers) available for use in all tests. Without this, I wouldn't be able to use matchers such as [`expect(...).toBeInTheDocument()`](https://github.com/newrelic/gatsby-theme-newrelic/pull/100/files#diff-c6211204be539afd5bf8ed94af3a1f1dR46)